lnrpc/routerrpc/router.proto

```diff
@@ -120,11 +120,23 @@ message SendPaymentRequest {
     amount.
     */
     uint32 max_shards = 17;
+
+    /**
+    If set, only the final payment update is streamed back. Intermediate updates
+    that show which htlcs are still in flight are suppressed.
+    */
+    bool no_inflight_updates = 18;
 }
 
 message TrackPaymentRequest {
     /// The hash of the payment to look up.
     bytes payment_hash = 1;
+
+    /**
+    If set, only the final payment update is streamed back. Intermediate updates
+    that show which htlcs are still in flight are suppressed.
+    */
+    bool no_inflight_updates = 2;
```

Comment:
... Or this `inflight_updates`

Reply:
The reason for doing `no_inflight_updates` is backwards compatibility. To not break even more on the rpc interface

Reply:
Yeah what I meant is that the commit was stale, now it's ok.